tests/phpunit/tests/media.php

```diff
@@ -3777,7 +3786,6 @@ public function test_wp_filter_content_tags_with_loading_optimization_attrs() {
 		// Following the threshold of 2, the first two content media elements should not be lazy-loaded.
 		$content_unfiltered = $img1 . $iframe1 . $img2 . $img3 . $iframe2;
 		$content_expected   = $prio_img1 . $iframe1 . $lazy_img2 . $lazy_img3 . $lazy_iframe2;
-		$content_expected   = wp_img_tag_add_decoding_attr( $content_expected, 'the_content' );
```

Comment:
I think we will be able to address the test failure here by adding `add_filter( 'wp_img_tag_add_decoding_attr', '__return_false' );` to the top of the function.